src/jacksettings.py

```diff
@@ -627,17 +629,21 @@ def loadDriverSettings(self, reset=False, forceReset=False):
             elif attribute == "capture":
                 if self.fDriverName == "firewire":
                     self.ui.obj_driver_capture.setCurrentIndex(1 if bool(value) else 0)
+                elif self.fDriverName == "dummy":
+                    self.setComboBoxValue(self.ui.obj_driver_capture, str(int(value)), True)
```

Comment:
this line is only relevant to dummy drivers? alsa should need the same treatment right?
makes more sense to always apply it, can you do that please?

Reply:
No because dummy driver is the only one where the Input and Output Devices are changed to Input and Output Ports, which are integers and not strings.

Reply:
ah alright, makes sense